__tests__/crowdfundings/payPledge.test.js

```diff
@@ -157,21 +167,22 @@ test('pay MONTHLY_ABO pledge with STRIPE', async (t) => {
 
   const periods = await pgDatabase().public.membershipPeriods.find({ membershipId: membership.id })
   t.ok(periods.length === 1, 'exactly 1 membership period generated')
+  t.ok(moment().add('1', 'month').isAfter(moment(periods[0].endDate)), 'membership ends after 1 month from now')
```

Comment:
add something like:
`t.ok(moment().add('25', 'days').isBefore(moment(periods[0].endDate)), 'membership does not end before 25 days from now')`